generator/search_index_builder.cpp

```diff
@@ -385,7 +385,7 @@ bool BuildSearchIndexFromDataFile(string const & filename, bool forceRebuild)
 {
   Platform & platform = GetPlatform();
 
-  FilesContainerR readContainer(platform.GetReader(filename));
+  FilesContainerR readContainer(platform.GetReader(filename, "f"));
```

Comment:
It fixes a bug when generator_tool crashes if there is a certain-named mwm file in `omim/data`. We didn't encounter it before because we don't have `minsk-pass` in the list of produced regions.